ci/linkcheck.sh

```diff
@@ -1,14 +1,20 @@
 #!/usr/bin/env bash
 
-if [ "$TRAVIS_EVENT_TYPE" = "cron" ] ; then
+if [ "$TRAVIS_EVENT_TYPE" = "cron" ] ; then # running in cron job
   FLAGS=""
 
   echo "Doing full link check."
-else
-  CHANGED_FILES=$(git diff-tree --no-commit-id --name-only -r origin/master... | tr '\n' ' ')
+elif [ "$CI" = "true" ] ; then # running in PR CI build
+  CHANGED_FILES=$(git diff-tree --no-commit-id --name-only -r $TRAVIS_COMMIT_RANGE | tr '\n' ' ')
   FLAGS="-f $CHANGED_FILES"
 
-  echo "Checking files changed in origin/master...: $CHANGED_FILES"
+  echo "Checking files changed in $TRAVIS_COMMIT_RANGE: $CHANGED_FILES"
+else # running locally
+  COMMIT_RANGE=master..
```

Comment:
```suggestion
  COMMIT_RANGE=$(git merge-base master HEAD)..
```

Reply:
Why?

Reply:
The change to use `...` means this should no longer be necessary.